testutils/gradle/testing.gradle

```diff
@@ -23,7 +23,7 @@ dependencies {
     testCompile "org.spockframework:spock-core:${spock_version}"
     testCompile "org.spockframework:spock-spring:${spock_version}"
     testCompile "org.springframework.batch:spring-batch-test:${spring_batch_version}"
-    testCompile "org.springframework:spring-test:${spring_version}"
+    testCompile "org.springframework:spring-test:3.2.9.RELEASE"
```

Comment:
ServiceMix didn't release an OSGi-wrapped version of it since the testing jar isn't meant to run in an OSGi environment.